morph_core/src/MimeTypes.re

```diff
@@ -0,0 +1,1003 @@
+let getExtension = filename => {
```

Comment:
there is an opam package that deals with mime types https://github.com/mirage/ocaml-magic-mime

Reply:
good catch! switched to that guy.